web/templates/settings/linkifier_settings_admin.hbs

```diff
@@ -3,40 +3,29 @@
 
         <p>
             {{#tr}}
-            Configure regular expression patterns that will be
-            automatically linkified when used in Zulip message bodies or
-            topics.  For example to automatically linkify commit IDs and
-            issue numbers (e.g. #123) to the corresponding items in a GitHub
-            project, you could use the following:
+            Configure regular expression patterns that will be used to
+            automatically transform any matching text in message topics
+            and content into links.
             {{/tr}}
         </p>
-        <ul>
-            <li>
-                <code>#(?P&lt;id&gt;[0-9]+)</code>
-                {{t "and" }}
-                <code>https://github.com/zulip/zulip/issues/%(id)s</code>
-            </li>
-            <li>
-                <code>(?P&lt;id&gt;[0-9a-f]{40})</code>
-                {{t "and" }}
-                <code>https://github.com/zulip/zulip/commit/%(id)s</code>
-            </li>
-        </ul>
         <p>
             {{#tr}}
-            Or, to automatically linkify GitHub's <code>org/repo#1234</code> syntax:
+            For example, to automatically linkify issue numbers (e.g. #123)
+            to the corresponding items in a GitHub project, you could use the
+            following regular expression pattern and URL format string:
             {{/tr}}
```

Comment:
Maybe we should just reuse the text on the help center page for the second paragraph here?

> Linkifiers make it easy to refer to issues or tickets in third party issue trackers, like GitHub, Salesforce, Zendesk, and others. For instance, you can add a linkifier that automatically turns #2468 into a link to https://github.com/zulip/zulip/issues/2468.

Reply:
Note that GitHub is linking to the issue here, but in the overlay it would just be text.

Reply:
See screenshot in [update post/comment](https://github.com/zulip/zulip/pull/24513#issuecomment-1450436639) below.

Reply:
Oh, I wasn't proposing cutting the bullets. Dropping the URL from the paragraph seems fine; I would cut "related" from your proposed text.